modal-go/client.go

```diff
@@ -371,6 +390,7 @@ func retryInterceptor() grpc.UnaryClientInterceptor {
 		inv grpc.UnaryInvoker,
 		opts ...grpc.CallOption,
 	) error {
+		c.logger.Debug("Sending gRPC request", "method", method)
```

Comment:
I tend to like `DebugContext` when a context is available. Is there any useful information in the context at this point?